lib/spack/spack/cmd/module.py

```diff
@@ -257,6 +257,18 @@ def module(parser, args):
     }
     query_args = constraint_qualifiers.get(args.subparser_name, {})
     specs = args.specs(**query_args)
+    if args.subparser_name == 'loads':
+        if len(specs) > 1:
+            message = ("Trying to load multiple modules, please be more "
+                       "specific. Currently we would load: \n\t" +
+                       "\n\t".join(str(s) for s in specs) + "\n")
```

Comment:
I added it specifically, so that the user (i.e. me) can be sure, that spack didn't actually change anything in the environment, I can remove it if you want, but since spack, at least right now, has the reputation of being overeager (at least at our lab), I'd prefer to be specific.